docs/tutorials/starting/analysis_1.ipynb

```diff
@@ -109,7 +109,8 @@
    "metadata": {},
```

Comment:
<p>I would not recall this here. This is a general point in all tutorials. Also the link to the GitHub repo for gammapy-data is not the regular way for users to access the data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
The last commit should have fix the problem